### ISSUE TITLE
option to suffix platform type to app name instead of prefixing

### ISSUE THREAD

**Issue by galki** [nstudio/xplat#54]
im talking about changing:
```
apps/
  ionic-myapp
  web-anotherapp
  web-myapp
```
to:
```
apps/
  anotherapp-web
  myapp-ionic
  myapp-web
```

prefixing and suffixing is basically grouping and im in favor of grouping by app instead of platform because:
1. its easier to use the directory tree when having dozens of apps and multiple platforms per app
2. my intuitive reasoning is ordered to prioritize the app over the platform (`start.app.platform.ios` instead of `start.platform.app.ios`)
3. if i understand correctly the current way conflicts with [nx](https://nrwl.io/nx/guide-nx-workspace):
```
ng g app myapp
ng g app myapp2 --directory=mydirectory
ng g lib mylib
ng g lib mylib2 --directory=mydirectory

ng build myapp
ng build mydirectory-myapp2
ng build mylib # work if the lib is marked as publishable
ng build mydirectory-mylib2 # work if the lib is marked as publishable
```

**Comment by KeithGillette** [comment]
I second @galki's recommendation. I like the `xplat` approach a lot but also think application-first, platform-second during development and would love to be able to organize the project accordingly.

**Comment by NathanWalker** [comment]
@galki I have this support coming in this [PR](https://github.com/nstudio/xplat/pull/75) now - just looking at one more thing related to web app external schematic which I should clear soon.

Again thank you for this feature request 👍

**Comment by NathanWalker** [comment]
This is now available in `7.2.5` - instructions of use: https://github.com/nstudio/xplat/wiki/How-to-group-by-appname-platform-instead%3F
